src/renderer/shared/ui-kit/Modal/Modal.tsx

```diff
@@ -53,6 +58,7 @@ const Root = ({ isOpen, size = 'md', height = 'fit', children, onToggle }: Props
               },
             )}
           >
+            {hasTitle ? null : <Dialog.Title hidden />}
```

Comment:
Hm, could you explain why is title - null when `hasTitle === true`?
By the way, `modalNodes.find(...)` could be changed to `modalNodes.some(...)` to get pure boolean value